### ISSUE TITLE
Negative coordinates appear during the test

### ISSUE THREAD

**Issue by zxin8218** [AlexeyAB/darknet#6422]
When I process a list of images and save results of detection to result.txt file, I find negative coordinates as follows:
`test/209.jpg: Predicted in 15.484000 milli-seconds.`
`ship: 74%	(left_x:  272   top_y:   -2   width:   63   height:   57)`
`ship: 97%	(left_x:  686   top_y:  648   width:   79   height:   45)`
Can anyone help me? Thanks

**Comment by stephanecharette** [comment]
Yes, sometimes the predictions are beyond the edges of the image.  This is "normal".

If you want, DarkHelp has a flag to ensure that all coordinates returned are capped within the image itself:  https://www.ccoderun.ca/darkhelp/api/classDarkHelp.html#ab3556e5fc64a38665b87d72df0746e6a

By default, this flag is enabled when using the DarkHelp C++ wrapper.
